src/explore-education-statistics-admin/src/pages/publication/components/PublicationManageTeamAccessTab.tsx

```diff
@@ -0,0 +1,54 @@
+import { BasicPublicationDetails } from '@admin/services/publicationService';
+import React from 'react';
+import Details from '@common/components/Details';
+import orderBy from 'lodash/orderBy';
+import ReleaseContributorPermissions from '@admin/pages/publication/components/ReleaseContributorPermissions';
+
+export interface Props {
+  publication: BasicPublicationDetails;
+}
+
+const PublicationManageTeamAccessTab = ({ publication }: Props) => {
+  if (
+    publication.releases === undefined ||
+    (publication.releases && publication.releases.length === 0)
+  ) {
+    return <p>No releases!</p>;
+  }
+
+  const sortedReleases = orderBy(publication.releases, r =>
+    parseInt(r.releaseName, 10),
+  );
+  const newestRelease = sortedReleases.shift();
+
+  if (newestRelease === undefined) {
```

Comment:
Changing to `if (!newestRelease)` will probably do away with need the question marks later down it the code.